Dockerfile

```diff
@@ -15,8 +15,8 @@ LABEL io.k8s.description="Base image for Red Hat Insights Compliance" \
 
 # Install dependencies and clean cache to make the image cleaner
 USER root
-RUN rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm && \
-    yum install -y openscap qt5-qtwebkit-devel jemalloc hostname && \
+RUN rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm && \
+    yum install -y jemalloc hostname && \
```

Comment:
We don't need `openscap` anymore as we don't use it anywhere, we parse with `https://github.com/openscap/openscap_parser` directly